pkg/Library/Shader/FragmentShader.js

```diff
@@ -8,7 +8,7 @@
   async initialize({}, state, {service}) {
     state.canvas = await service({kind: 'ThreejsService', msg: 'allocateCanvas'});
   },
-  async update(inputs, state, {service}) {
+  async update(inputs, state, {service, invalidate}) {
```

Comment:
this is unused

Reply:
Ty. Fwiw, I was experimenting with the Shader invalidating itself as a way around the 'no clock' problem of shading static images.